src/Components/TopicsCategory.jsx

```diff
@@ -1,34 +1,36 @@
-// import { useState, useEffect } from "react"
-// // import { Link } from "react-router-dom"
-// import { fetchArticles } from "../../utils"
-// import { useParams } from "react-router-dom"
+import { useState, useEffect } from "react"
+import { fetchArticles } from "../../utils"
+import { useParams } from "react-router-dom"
+import { Link } from "react-router-dom"
 
 
-// function TopicsCategory() {
+function TopicsCategory() {
 
-// const [articles, setArticles] = useState([])
-// const {slug} = useParams()
+const [articles, setArticles] = useState([])
+const { topic } = useParams()
 
-// useEffect(() => {
-//     fetchArticles().then(({articles}) => {
-//         setArticles(articles)
-//     })
-// }, [])
+useEffect(() => {
+    fetchArticles(topic).then(({articles}) => {
+        setArticles(articles)
+    })
+}, [topic])
 
-// return (
-//     <>
-//         <main>
-//             <ul>{articles.map((article) => {
-//                 if (article.topic === slug) {
-//                     return <li key={article.article_id}>
-//                             <img src={article.article_img_url} />
-//                         <h2>{article.title}</h2>
-//                     </li>
-//                 }
-//             })}</ul>
-//         </main>
-//     </>
-// )
-// }
+return (
```

Comment:
Indicating that the articles (by topic) are loading would be nice. Same on the 'Topics' page when loading the topics.